.gitignore

```diff
@@ -2,5 +2,5 @@
 .vscode
 # turn off these for now
 specs
-vendor/**/examples
-vendor/**/*_test.go
\ No newline at end of file
+#vendor/**/examples
```

Comment:
this is need because go build all f*cking source code in the folders and there are only examples... that need nobody in a vendor system...